resources/lang/hu/localizations.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
+    'change_not_hu_en' => 'Válassz egy nyelvet, ami nem angol, vagy magyar!',
+    'help_translate' => 'Segíts nekünk fordítani!',
+    'help_translate_info' => 'Válassz egy nyelvet, amiben szeretnél segíteni fordítani. Ha látsz egy kifejezést, ami nem helyes azon a nyelven, javítsd ki, küldd el, és ha egy rendszergazda elfogadja a javításodat, akkor megjelenik az az :appban is! (Csak azok a kifejezések láthatóak, amelyekhez nem tartozik felülvizsgálatra váró változtatás.)',
```

Comment:
nit: you could write "adminisztrátor" instead of "rendszergazda" since not only sysadmins will check these.